server/src/main/java/keywhiz/service/daos/SecretDeletionMode.java

```diff
@@ -0,0 +1,6 @@
+package keywhiz.service.daos;
+
+public enum SecretDeletionMode {
```

Comment:
WDYT about a fromString, or something similar, static method that that defers to EnumUtils?